MapboxDirections/MBRouteStep.swift

```diff
@@ -179,27 +179,41 @@ public enum ManeuverType: Int, CustomStringConvertible {
     case useLane
      
     /**
-     The step requires the user to enter, traverse, and exit a roundabout (traffic circle or rotary).
+     The step requires the user to enter and traverse a roundabout (traffic circle or rotary).
      
      The step has no name, but the exit name is the name of the road to take to exit the roundabout. The exit index indicates the number of roundabout exits up to and including the exit to take.
      */
     case takeRoundabout
     
     /**
-     The step requires the user to enter, traverse, and exit a large, named roundabout (traffic circle or rotary).
+     The step requires the user to enter and traverse a large, named roundabout (traffic circle or rotary).
      
      The step’s name is the name of the roundabout. The exit name is the name of the road to take to exit the roundabout. The exit index indicates the number of rotary exits up to and including the exit that the user must take.
```

Comment:
Add this paragraph:

> If `RouteOptions.includeExitRoundaboutManeuver` is set to `true`, this step is followed by an `.exitRotary` maneuver. Otherwise, this step represents the entire roundabout maneuver, from the entrance to the exit.